examples/lighting-app/bouffalolab/common/AppTask.cpp

```diff
@@ -644,18 +644,19 @@ void AppTask::ButtonEventHandler(void * arg)
     uint32_t presstime;
     if (ButtonPressed())
     {
-        hosal_gpio_irq_set(&gpio_key, HOSAL_IRQ_TRIG_NEG_LEVEL, GetAppTask().ButtonEventHandler, NULL);
+        // hosal_gpio_irq_set(&gpio_key, HOSAL_IRQ_TRIG_NEG_LEVEL, GetAppTask().ButtonEventHandler, NULL);
```

Comment:
Maybe you should consider removing these commented lines, or at leas add proper comment why it's commented out.

Reply:
removed